pkg/eks/eks.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/kris-nova/logger"
 	"github.com/pkg/errors"
 
+	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/aws/aws-sdk-go/service/eks"
 	awseks "github.com/aws/aws-sdk-go/service/eks"
```

Comment:
Not a problem with this branch specifically, but lines 20 & 21 are importing the same package?

Reply:
lol, totally didn't see that while copy-pasta from other pr